<ui version="4.0" >
 <class>ChatWindowConfig_Style</class>
 <widget class="QWidget" name="ChatWindowConfig_Style" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>417</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Chat Window Appearance</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Display</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="kcfg_groupConsecutiveMessages" >
          <property name="text" >
           <string>Group consecuti&amp;ve messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_useCompact" >
          <property name="text" >
           <string>Co&amp;mpact style in chatrooms</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="stylesGroupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Styles</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Style variant:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="variantList" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" >
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible" >
           <bool>false</bool>
          </property>
          <widget class="QListWidget" name="styleList" />
          <widget class="QFrame" name="htmlFrame" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0" >
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="KPushButton" name="btnGetStyles" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>Get new Chat Window styles over the Internet</string>
            </property>
            <property name="text" >
             <string>&amp;Get New...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="installButton" >
            <property name="text" >
             <string>&amp;Install...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="deleteButton" >
            <property name="text" >
             <string>&amp;Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>styleList</tabstop>
  <tabstop>variantList</tabstop>
  <tabstop>btnGetStyles</tabstop>
  <tabstop>installButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>kcfg_groupConsecutiveMessages</tabstop>
  <tabstop>kcfg_useCompact</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
